configuration/introduction.md

```diff
@@ -1,47 +1,119 @@
-| `electronSwitches` | An optional array of Electron switches. This allows configuration of electron app itself. <br> This properties will not affect the `serveronly` mode. Usually normal `MM` users don't need this property, but if you are a hard-core hacker, you might need this to handle Electron itself over `MagicMirror` provides. More options can be found [here](https://www.electronjs.org/docs/latest/api/command-line-switches) (Not all available switches are described there.)<br>example:`electronSwitches:["enable-transparent-visuals", "disable-gpu"];` |
-| `customCss`        | The path of the `custom.css` stylesheet. The default is `css/custom.css`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
-
-### Environment variables
-
-As the `config.js` is javascript code there is no direct way to provide
-environment variables. This feature was requested by some users, e.g. with
-environment variables it is easier to share the configuration with others
-(otherwise you have to delete all private data as e.g. api keys). There was a
-longer
-[discussion on github](https://github.com/MichMich/MagicMirror/issues/1756)
-where you can read the full story.
-
-#### `config.js` and `config.js.template`
-
-You can provide a `config.js.template` instead of a `config.js` file. The
-difference is, that you can use environment variables in the
-`config.js.template`. When starting MagicMirror² a `config.js` is created from
-`config.js.template` and the variables are resolved.
+
+
+### Extremely Basic Configuration of MagicMirror²: 
+
+1. Go to your configuration directory: In Bash/linux, it'll be in `~/MagicMirror/config/`
+
+2. create and edit a config.js by using your editor of choice. 
+
+3. Minimum configuration requires the following: 
+
+   ``` js
+   let config = {
+     modules: [
+       {
+           module: clock,
+           position: fullscreen_above,
+           config: {
+       	}
+       },
+     ],
+   };
+   ```
+
+   See [module configuration](/modules/configuration.md) for more information and examples. 
+
+4.  run your magic mirror. 
```

Comment:
Would a note on how to run it be useful here?

Reply:
There's too many ways to run it, and it's covered previously in the documentation, so I didn't include it here.

Reply:
Ah, okay. Perhaps a link or reference where it is described?

Reply:
ooh, good call. Adjusted.

Reply:
Nice! Thanks :slightly_smiling_face: